library-best-practices/JLBP-12.md

```diff
@@ -0,0 +1,8 @@
+[JLBP-12] Make level of support and level of API stability clear
```

Comment:
level of support and API stability

Reply:
I couldn't decide on this one - you're the tie breaker :-)